web-basics-browser-security-model/activities/cockroach/README.md

```diff
@@ -0,0 +1,19 @@
+# Name
```

Comment:
Why is this a big header (`#`) and the others are small (`##`)?

Reply:
Legacy, I don't know any reason for it.